spec/features/job_seekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -91,5 +125,40 @@
         expect(page).to have_content('You are already subscribed to a daily email with these search criteria')
       end
     end
+
+    scenario 'can succesfuly subscribe to a new alert', elasticsearch: true do
+      create_list(:vacancy, 5, :published, job_title: 'Math')
+      create_list(:vacancy, 3, :published, job_title: 'English')
+
+      Vacancy.__elasticsearch__.client.indices.flush
+
+      visit jobs_path
+
+      within '.filters-form' do
+        fill_in 'keyword', with: 'English'
+        page.find('.govuk-button[type=submit]').click
```

Comment:
On the subscription new view we could move the 'Subscribe' button text to locale and use that here to remove our coupling on the way the DOM is specifically set out:

```
click_on(locale.for.subscribe.button)
```